### PR TITLE
TASK: Improve SonarQube Code Maintability - High Severity

### PR DESCRIPTION
**Pull request by enyc24**
## Description:
Improves code maintanability of high severity levels by refactoring methods to reduce code complexity by extracting helper methods and making constants. Note that the googlemaps_livelocation.dart file has not been touched to avoid merge conflicts. 

## Related User Story:
https://github.com/Mini-Capstone-Fried-Chicken/Fried-Chicken/issues/176

## Screenshot/Demo:
N/A

## How to test:
N/A 

## Type of Change:
- [ ] Feature
- [ ] Bug fix
- [✖ ] Refactor
- [ ] Documentation update
- [ ] Tests
- [ ] Others

## Pull Request Checklist
- [x] Flutter CI test passed
- [x] Sonar test passed
- [x] Codecov test passed
- [✖ ] Application runs without errors
- [✖ ] Feature tested locally on a high level
- [✖ ] Architecture conventions respected
- [ ✖ ] No API keys or secrets committed
- [ ✖ ] Document when necessary

## PR Review Checklist (dev)
- [ ✖ ] Code is readable and well structured
- [ ✖ ] No obvious bugs or edge cases missed
- [ NA] Tests are adequate
- [✖ ] Naming and organization are clear
- [✖ ] Feature works as described
- [✖ ] No unnecessary complexity
- [✖ ] Follows project architecture

## PR Review Checklist (QA)
- [x] Made sure to test the application fully
- [x] No bugs created by this PR

### CODE REVIEW

File: lib/features/auth/ui/forgot_password_page.dart [hunk 1]
```diff
@@ -114,7 +114,7 @@ class _ForgotPasswordState extends State<ForgotPassword> {
                         ),
                       ),
                     ),
-                    Align(
+                    const Align(
```
Comment (HoangVuLuu) [inline lib/features/auth/ui/forgot_password_page.dart]: I am not sure to understand why did you make it a ''Const''

**Review by HoangVuLuu — COMMENTED** [review]
Inside indoor_page.dart, I see that the first 400 lines of code is commented out, can we remove that? Same thing for the file google_places_service.dart
